tests/config/indent_ctor_init.cfg

```diff
@@ -1,1289 +1,18 @@
-# Uncrustify 0.58
```

Comment:
I see, that's a great systematic way to do it! Yeah, the only differences I see are

- `indent_with_tabs`, which I just add always for good measure, even though it does not make a difference here, and

- my `nl_collapse_empty_body` vs. your `nl_class_leave_one_liners`, and there's probably no harm in leaving my stronger option in.

Thanks for the verification.